src/cloudWatchLogs/commands/searchLogGroup.ts

```diff
@@ -60,8 +60,13 @@ function handleWizardResponse(response: SearchLogGroupWizardResponse, registry:
     return initialStreamData
 }
 
-export async function prepareDocument(uri: vscode.Uri, registry: LogStreamRegistry): Promise<telemetry.Result> {
+export async function prepareDocument(
+    uri: vscode.Uri,
+    initialLogData: CloudWatchLogsData,
+    registry: LogStreamRegistry
+): Promise<telemetry.Result> {
     try {
+        await registry.registerLog(uri, initialLogData)
```

Comment:
This works but hopefully you can see why restricting this 'try/catch' block to `prepareDocument` is problematic. Don't change anything of course, just understand that this fix was for _one_ bug.